server/src/controller/authControllers.ts

```diff
@@ -155,9 +156,74 @@ const login = async (req: Request, res: Response) => {
   }
 };
 
+const sendPasswordResetEmail = async (req: Request, res: Response) => {
+  try {
+    const {email}=req.body;
+    
+    const result = await db.pool.query(
+      "SELECT email FROM public.users WHERE email = $1",
+      [email]
+    );
+   
+    if(result.rowCount==0){
```

Comment:
Now you can check rows[0] or rows.length > 0 condition

Reply:
Any issues if I use that???

Reply:
There is no issue but it's better to have less and consistent code